static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,121 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Tracked data is stored in a **centralized** remote location, with the ability
+  to create distributed copies on other remotes.
+- Several projects can **share** the same files, guaranteeing that everyone has
+  access to the same data versions. See
+  [Share Data and Model Files](/doc/use-cases/share-data-and-model-files) for
+  more information.
+- Projects that import data from the registry don't need to push these large
+  files to their own [remotes](/doc/command-reference/remote), **saving space**
+  on storage – they may not even need a remote at all, using only their local
+  <abbr>cache</abbr>.
+- DVC data registries can handle multiple versions of data and ML modes with a
+  familiar CLI. See
+  [Data and Model Files Versioning](/doc/use-cases/data-and-model-files-versioning)
+  for more information.
+- DVC data registries are versioned with Git, so you can always track the
+  history of the project the same as you manage your source code repository.
+
+A possible risk of shared data registries is that, if the source project or its
+remote storage are lost for any reason, several other projects depending on it
+may stop being reproducible. So this strategy is best when the registry is owned
+and controlled internally by the same team as the projects that employ it.
+
+## Using properly versioned registries
```

Comment:
rename it to "Example"? it's just not clear are there some registries that are not properly versioned? why btw we version registries not data in them? etc..

Reply:
I would start with a wider introduction here - something like:

`Let's take as an example these docs you are reading now. We have a lot of examples, tutorials, etc that use different datasets and produce different models. Some of these datasets are being reused across multiple example or projects. We update them from time to time the same way we update code of our examples. It was clearly a first candidate to introduce a data registry using DVC repository.`

Reply:
H2 renamed. I'll follow up about the wider into in https://github.com/iterative/dvc.org/pull/679#pullrequestreview-304977884.